common/src/main/java/com/microsoft/identity/common/internal/ui/webview/challengehandlers/ClientCertAuthChallengeHandler.java

```diff
@@ -165,22 +173,18 @@ public void run() {
                             synchronized (sDeviceLock) {
                                 Logger.verbose(TAG, "A YubiKey device was disconnected");
                                 mDevice = null;
+                                //Creating a CertBasedAuthEvent for Telemetry.
+                                final CertBasedAuthEvent certBasedAuthEvent = new CertBasedAuthEvent(TelemetryEventStrings.Event.CERT_BASED_AUTH_SMARTCARD_EVENT);
```

Comment:
Double check with shahzaib, but i'm wondering if we should just create a new event type specifically for this

i.e.  TelemetryEventStrings.Event.YUBIKEY_PROVIDER_STATUS_EVENT

Reply:
Could it potentially be a new event name instead? So that it's under the event type CERT_BASED_AUTH_EVENT but the event name is specifically CERT_BAED_AUTH_YUBIKEY_PROVIDER_STATUS_EVENT? (idk maybe that's too long)